components/mpas-seaice/cime_config/buildnml

```diff
@@ -307,6 +307,15 @@ def buildnml(case, caseroot, compname):
                 grid_date = '20231121'
                 grid_prefix = 'mpassi.IcoswISC30E3r5.rstFromG-chrysalis'
 
+    elif ice_grid == 'RRS6to18E3r6':
+        decomp_date = '20240402'
+        decomp_prefix = 'partitions/mpas-seaice.graph.info.'
+        grid_date = '20240402'
+        grid_prefix = 'mpassi.RRS6to18E3r6'
+        if ice_ic_mode == 'spunup':
+            logger.warning("WARNING: The specified compset is requesting seaice ICs spunup from a G-case")
+            logger.warning("         But no file available for this grid.")
```

Comment:
@darincomeau, I didn't include the DIB file here but Compass produces one.  Do you ever anticipate wanting DIB files for simulations without cavities?  They're easy to produce but should I leave them out just for cleanliness when we don't have cavities?

Reply:
I agree to leave them out when we don't have cavities - we don't really have current compsets that would use DIB without some form of ISMF (except the older JRA-DIB only ones, I don't think we did it for JRA1p5).

Reply:
Okay, I've turned off generating DIB files in meshes without ice-shelf cavities for now.